radiomics/firstorder.py

```diff
@@ -30,6 +30,12 @@ def __init__(self, inputImage, inputMask, **kwargs):
     self.pixelSpacing = inputImage.GetSpacing()
     self.voxelArrayShift = kwargs.get('voxelArrayShift', 0)
 
+    self._initLesionWiseCalculation()
+
+  def _initLesionWiseCalculation(self):
+    super(RadiomicsFirstOrder, self)._initLesionWiseCalculation()
+    self.targetVoxelArray = self.imageArray[self.ROICoordinates].astype('float')
+
     self.logger.debug('Feature class initialized')
```

Comment:
For consistency, may be more helpful to have logging message `<feature class name> feature class initialized` for each of the feature classes?